specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/streamingservice.json

```diff
@@ -1143,7 +1143,7 @@
           ],
           "x-ms-enum": {
             "name": "LiveOutputResourceState",
-            "modelAsString": false
+            "modelAsString": true
```

Comment:
This introduces a breaking change in a stable swagger, is it intentional?

Reply:
Yes, this is intentional. All other enumerations in our API were modelAsString in our first V3 submission. https://github.com/Azure/azure-rest-api-specs/pull/4483. Anu and Sameer agreed that this is not a breaking change.

Reply:
It's a breaking change from an SDK perspective.  We won't block on it/require an updated API version but it will impact some customers (e.g. C# codegen is quite different).